extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -710,10 +659,112 @@ export const BlockaidTxScanLabel = ({
   scanResult,
   onClick,
 }: {
-  scanResult: BlockAidScanTxResult;
+  scanResult: BlockAidScanTxResult | null | undefined;
   onClick: () => void;
 }) => {
   const { t } = useTranslation();
+  const shouldTreatAsUnableToScan = useShouldTreatTxAsUnableToScan();
+  const isTxSuspiciousCheck = useIsTxSuspicious();
+  const [blockaidState, setBlockaidState] = useState<string | null>(null);
```

Comment:
I would rename these vars to `[blockaidOverrideState, setBlockaidOverrideState` so they're more descriptive of what the values are being used for